TLM/TLM/U/UBuilder.cs

```diff
@@ -17,13 +17,14 @@ namespace TrafficManager.U {
     public class UBuilder {
         public AtlasBuilder AtlasBuilder;
```

Comment:
why is this a field and public?
is there a place you want to override it after instantiation?
if you can set it via ctor, it should be sufficient I assume.